[5.11.0] Add and update STS content (#3684)
wso2/docs-is: en/docs/learn/configuring-ws-trust-security-token-service.md

# Configuring WS-Trust Security Token Service

WSO2 Identity Server uses the security token service (STS) as the [WS-Trust](../../learn/ws-trust) implementation.

The STS is capable of issuing SAML 1.1 and 2.0 security tokens and has a SOAP/XML API for token issuance.

## Download the STS connector

> WS-Trust authentication is no longer supported by default from IS 5.11.0 upwards and has been introduced as a connector. In order to use WS-Trust authentication, first, you should download the connector from WSO2 Connector Store.

Reviewer: ```suggestion
WS-Trust authentication is no longer supported by default from IS 5.11.0 upwards and has been introduced as a connector. In order to use WS-Trust authentication, first, you should download the connector from the WSO2 Connector Store.
```

    <td><code>localhost</code></td>
    </tr>
    <tr class="even">
    <td><strong>Idle Session Time Out</strong></td>
    <td>This is the duration in minutes for which an SSO session can be idle. If WSO2 Identity Server receives no SSO authentication requests for the given duration, a session time-out occurs. The default value is <code>15</code> .</td>
    <td><code>15</code></td>
    </tr>
    <tr class="odd">
    <td><strong>Remember Me Period</strong></td>
    <td><div class="content-wrapper">
    <p>This is the duration in weeks for which WSO2 Identity Server should remember an SSO session given that the <strong>Remember Me</strong> option is selected in the WSO2 Identity Server login screen.</p>
    <p>The default value is <code>2</code> weeks.</p>
    </div></td>
    <td><code>2</code></td>
    </tr>
    </tbody>
    </table>

3. Expand **Inbound Authentication Configuration** > **Security Token Service Configuration** and click **Apply Security Policy**.
    ![security-token-service-config](../assets/img/tutorials/security-token-service-config.png)

> 4. Select `Yes` from the **Enable Security?** list and select a pre-configured security scenario according to your requirements.

Reviewer: ```suggestion
4. Select **Yes** from the **Enable Security?** list and select a pre-configured security scenario according to your requirements.
```

We will be using `UsernameToken` under the **Basic Scenarios** section for this tutorial.  
    ![enable-security](../assets/img/tutorials/enable-security.png)

    !!! note
        You can find further details about security policy scenarios from the **view scenario** option.
        ![security-policy-scenarios](../assets/img/tutorials/security-policy-scenarios.png)

5. Click **Next**.

    !!! info

>         The next steps may vary per the security scenario you have chosen under point (5) above. Below is for **UsernameToken** scenario.

Reviewer: ```suggestion
        The next steps may vary according to the security scenario you have chosen under point (5) above. Below is for the **UsernameToken** scenario.
```

6. Provide the required details as follows:

>     1. Select `ALL-USER-STORE-DOMAINS` as the domain.

Reviewer: ```suggestion
    1. Select **ALL-USER-STORE-DOMAINS** as the domain.
```
